### ISSUE TITLE
Windows: Run swift script command

### ISSUE THREAD

**Issue by adam-fowler** [swift-server/vscode-swift#164]
Verify `Run Swift Script` command works. Commands are displayed with Ctrl+Shift+P

There are two situations to verify here
1) Where you have a new file with type set to swift but it is not saved
2) A swift file that is already saved
Verify

- [ ] LSP server works. You should symbol completion for variables local to the file and the stdlib, if you import Foundation you should see symbol completion for Foundation.
- [ ] The swift script runs as expected. If it hasn't been saved it is saved to a temporary file and that file is run. After it has completed the temporary file should have been deleted

**Comment by adam-fowler** [comment]
These are the options detailed on swift.org for running the REPL
```
set SWIFTFLAGS=-sdk %SDKROOT% -I %SDKROOT%/usr/lib/swift -L SDKROOT%/usr/lib/swift/windows
swift repl -target x86_64-unknown-windows-msvc %SWIFTFLAGS%
```

**Comment by stevapple** [comment]
@adam-fowler Do you know if there’s a way to detect if `swift` is using the new driver — I’d like to solve this by optionally appending necessary flags with old driver (C++) only.

**Comment by adam-fowler** [comment]
> @adam-fowler Do you know if there’s a way to detect if `swift` is using the new driver — I’d like to solve this by optionally appending necessary flags with old driver (C++) only.

Can we do it based on Swift version?

**Comment by stevapple** [comment]
Makes sense with official releases, but not so ideal.

**Comment by adam-fowler** [comment]
On macOS `swift --version` also outputs a swift-driver version to stderr. Use of stdout and stderr if totally weird with macOS tools. Doesn't seem to output driver version on Linux

**Comment by stevapple** [comment]
When I tried to solve this, it occurs that using `swift` to run a script is completely broken on Windows — that is, the only way we can run a script is to compile it with `swiftc` and run the executable directly.

Do you think we still need to make it possible, or we just hide the task from Windows users with older toolchain.

**Comment by adam-fowler** [comment]
> Do you think we still need to make it possible, or we just hide the task from Windows users with older toolchain.

It's easy enough to hide the functionality, but it might be better to display a notification saying it isn't available on older versions of swift. 

Are you saying the latest Windows toolchain can run swift scripts?

**Comment by stevapple** [comment]
> Are you saying the latest Windows toolchain can run swift scripts?

Requires Swift 5.7, which is the next release.

**Comment by stevapple** [comment]
I implemented #253 on macOS, and will switch to Windows to verify that it works. This PR will not create `Run script` command on toolchains without Swift driver.

**Comment by adam-fowler** [comment]
Going to close this now
